packages/paste-core/components/spinner/src/styled.tsx

```diff
@@ -0,0 +1,27 @@
+import {styled} from '@twilio-paste/styling-library';
+
+import {SvgKeyframes, CircleKeyframes} from './keyframes';
+import {circleCircumference} from './constants';
+
+export const StyledCircleTrack = styled.circle(() => {
+  return {
+    transformOrigin: 'center',
+    opacity: 0.25,
+  };
+});
+
+export const AnimatedStyledCircle = styled.circle<{show: boolean}>(({show}) => {
+  return {
+    transformOrigin: 'center',
+    animation: `${1.5}s ease-in-out infinite both ${CircleKeyframes}`,
+    strokeDasharray: circleCircumference + 1,
```

Comment:
Can we leave a comment about why we need the `+1`?  We also have the `strokeDashArray` export in the constant file you can use